index.html

```diff
@@ -129,11 +129,10 @@ <h2 class="animated fadeIn">Contributors</h2>
 <!--End of contributors section-->
 
 <!-- Thanks -->
-<section class="bg-dark">
+<section class="bg-dark" style="height: 264px;">
```

Comment:
instead of using inline texts can you use bootstrap specific?

Reply:
Hi, I'm sorry I'm not that comfortable with Bootstrap. 
Actually I had to add this just so that the blinking cursor appears right after the text, to create the typing.
Also, this is just some custom inline CSS, which will never interfere with the regular Bootstrap framework.

Reply:
Hi, I totally understand your point. However, I had to use two occurences of inline styles, for the following reasons:
In `<section class="bg-dark" style="height: 264px;">`, I had to do it because when typed.js removes the text, the container gets resized for a moment and expands again. This was necessary using inline styles since I needed a fixed static height.

And for the `<h2 class="typed" style="display: inline-block;">`, it's just there to make sure the blinking cursor comes right after the text and doesn't show up in a new line, since most browsers' user agent stylesheet styles the heading elements (`h1` to `h6`).

I can say from past experience that none of these will ever interfere with the Bootstrap styling of the rest of the page, since that's the beauty of inline CSS. It also won't create any trouble with Bootstrap since Bootstrap is anyway CSS and JS.